static/css/style.css

```diff
@@ -240,28 +240,24 @@ nav {
     border-bottom: #222222 solid 2px;
 }
 
-.navigation {
-
-}
-
 .menuItem {
     height: 20px;
     display: inline;
     list-style: none;
-    padding: 0px 25px 7px 25px;
+    padding: 0px 25px 6px 25px;
     margin: 0px;
     background-image: url(../img/list_stroke.png);
     background-repeat: no-repeat;
-    background-position:bottom right;
+    background-position: bottom right;
+		position: relative;
 }
 
 .menuItem:last-child  {
     background-image: none;
 }
 
-li.menuItem > a,

```

Comment:
by deleting this ">a" we now have underlines on navigation items .. we should fix that (using firefox)